### PR TITLE
Devops: get gqcp_dev local and on HPC clusters

### PR DESCRIPTION
**Pull request by guacke**
**Short description**
This PR will ensure that developers can install GQCP locally and on the HPC clusters.

**Related issues**

**Additional context**

### CODE REVIEW

**Comment by guacke** [comment]
@lelemmen @dariatols This PR details installations on the HPC.

File: INSTALL.md [hunk 1]
```diff
@@ -5,15 +5,19 @@
...
 After installation, set the `LIBINT_DATA_PATH` environment variable to the folder that contains the libint bases. In a default installation (of e.g. version v2.3.1), the data path is given by:
 
 ```bash
-export LIBINT_DATA_PATH=${conda_install_dir}/share/libint/2.3.1/basis
+export LIBINT_DATA_PATH=$CONDA_PREFIX/share/libint/2.3.1/basis
 ```
 
+You will have to either export this environment variable everytime you activate the `gqcp` environment or (better) put this export in your .bashrc or (preferred) [add this environment variable to your virtual environment](https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html#saving-environment-variables).
+
 ## Develop install
```
Comment (lelemmen) [inline INSTALL.md]:
```suggestion
## Installation instructions for developers
```

Could you also adjust the header for the installation instructions for users?

File: INSTALL.md [hunk 2]
```diff
@@ -62,27 +68,23 @@ The possible CMake options are listed below. As such, for the provided GQCG envi
...
-
-which should return `${conda_install_dir}/bin/conda`.
-
 #### CMake options
 
 In general, please set and pass the following options to the `cmake ..` command:
 
+* `-DCMAKE_C_COMPILER=cc`, with `cc` the C-compiler used. On the HPC clusters, you can use the Intel compiler `icc` after loading the relevant module (e.g. `module load intel/2019b`).
+
+* `-DCMAKE_CXX_COMPILER=cxx`, with `cxx` the C++-compiler used. On the HPC clusters, you can use the Intel compiler `icpc` after loading the relevant module (e.g. `module load intel/2019b`).
```
Comment (lelemmen) [inline INSTALL.md]: I feel like these lines belong in specific instructions for the UGent HPC.

File: INSTALL.md [hunk 3]
```diff
@@ -114,3 +116,9 @@ For this library, there are several extra options and configuration arguments yo
 * `-DPYTHON_EXECUTABLE=python_executable` with `python_executable` the path to your preferred Python executable.
 
 * `-DPYTHON_LIBRARY=python_library` with `python_library` the path to the libraries that support your preferred Python executable.
+
+### Guidelines
+
+* If you encounter an EMT-instruction failure when using the Intel compilers on the HPC clusters, reduce the number of processors passed to make, e.g. replace `make -j6` by `make -j3`.
+* Store and compile the code on `$VSC_SCRATCH` to achieve optimal performance.
+
```
Comment (lelemmen) [inline INSTALL.md]: idem

**Pull request merged**